bin/MadGraph5_aMCatNLO/cards/production/13TeV/Wgamma_Monophoton_Ptg130_NLO_fxfx/Wgamma_Monophoton_Ptg130_NLO_run_card.dat

```diff
@@ -0,0 +1,152 @@
+  306000 = lhaid 
+#***********************************************************************
+# Include the NLO Monte Carlo subtr. terms for the following parton    *
+# shower (HERWIG6 | HERWIGPP | PYTHIA6Q | PYTHIA6PT | PYTHIA8)         *
+# WARNING: PYTHIA6PT works only for processes without FSR!!!!          *
+#***********************************************************************
+  PYTHIA8       =  parton_shower
+   1.0  =  shower_scale_factor  ! multiply default shower starting
+                                  ! scale by this factor
+#***********************************************************************
+# Renormalization and factorization scales                             *
+# (Default functional form for the non-fixed scales is the sum of      *
+# the transverse masses of all final state particles and partons. This *
+# can be changed in SubProcesses/set_scales.f)                         *
+#***********************************************************************
+  False =  fixed_ren_scale   ! if .true. use fixed ren scale
+  False =  fixed_fac_scale   ! if .true. use fixed fac scale
+  91.188        =  muR_ref_fixed     ! fixed ren reference scale
+  91.188        =  muF1_ref_fixed    ! fixed fact reference scale for pdf1
+  91.188        =  muF2_ref_fixed    ! fixed fact reference scale for pdf2
+  -1    =  dynamical_scale_choice  ! Choose one of the preselected dynamica
+#***********************************************************************
+# Renormalization and factorization scales (advanced and NLO options)  *
+#***********************************************************************
+  False =  fixed_QES_scale   ! if .true. use fixed Ellis-Sexton scale
+  91.188        =  QES_ref_fixed     ! fixed Ellis-Sexton reference scale
+  1.0   =  muR_over_ref      ! ratio of current muR over reference muR
+  1.0   =  muF1_over_ref     ! ratio of current muF1 over reference muF1
+  1.0   =  muF2_over_ref     ! ratio of current muF2 over reference muF2
+  1.0   =  QES_over_ref      ! ratio of current QES over reference QES
+#***********************************************************************
+# Reweight flags to get scale dependence and PDF uncertainty           *
+# For scale dependence: factor rw_scale_up/down around central scale   *
+# For PDF uncertainty: use LHAPDF with supported set                   *
+#***********************************************************************
+  True  =  reweight_scale    ! reweight to get scale dependence
+  0.5   =  rw_Rscale_down    ! lower bound for ren scale variations
+  2.0   =  rw_Rscale_up      ! upper bound for ren scale variations
+  0.5   =  rw_Fscale_down    ! lower bound for fact scale variations
+  2.0   =  rw_Fscale_up      ! upper bound for fact scale variations
+  False  =  reweight_PDF      ! reweight to get PDF uncertainty
+#  292201        =  PDF_set_min       ! First of the error PDF sets
+#  292302        =  PDF_set_max       ! Last of the error PDF sets
```

Comment:
Remove unnecessary lines

Reply:
Removed 
#  292201        =  PDF_set_min       ! First of the error PDF sets
#  292302        =  PDF_set_max       ! Last of the error PDF sets